tests/fast_int.cpp

```diff
@@ -0,0 +1,565 @@
+                                                              "-1000000000000000000001",
+                                                              "67404283172107811828",
+                                                              "-67404283172107811830",
+                                                              "9223372036854775808",
+                                                              "-9223372036854775809",
+                                                              "1728002635214590698",
+                                                              "-1728002635214590699",
+                                                              "41A792678515120368",
+                                                              "-41A792678515120369",
+                                                              "10B269549075433C38",
+                                                              "-10B269549075433C39",
+                                                              "4340724C6C71DC7A8",
+                                                              "-4340724C6C71DC7A9",
+                                                              "160E2AD3246366808",
+                                                              "-160E2AD3246366809",
+                                                              "8000000000000000",
+                                                              "-8000000000000001",
+                                                              "33D3D8307B214009",
+                                                              "-33D3D8307B21400A",
+                                                              "16AGH595DF825FA8",
+                                                              "-16AGH595DF825FA9",
+                                                              "BA643DCI0FFEEHI",
+                                                              "-BA643DCI0FFEEI0"
+                                                              "5CBFJIA3FH26JA8",
+                                                              "-5CBFJIA3FH26JA9",
+                                                              "2HEICIIIE82DH98",
+                                                              "-2HEICIIIE82DH99",
+                                                              "1ADAIBB21DCKFA8",
+                                                              "-1ADAIBB21DCKFA9",
+                                                              "I6K448CF4192C3",
+                                                              "-I6K448CF4192C4",
+                                                              "ACD772JNC9L0L8",
+                                                              "-ACD772JNC9L0L9",
+                                                              "64IE1FOCNN5G78",
+                                                              "-64IE1FOCNN5G79",
```

Comment:
I haven't checked these numbers, but are some of them just outside the range as well (like int64_min - 1 and int64_max + 1)?